test/runner/lib/cloud/vcenter.py

```diff
@@ -62,10 +79,16 @@ def setup(self):
         """Setup the cloud resource before delegation and register a cleanup callback."""
         super(VcenterProvider, self).setup()
 
-        if self._use_static_config():
-            self._setup_static()
-        else:
-            self._setup_dynamic()
+        # TODO mostly thinking out loud here, to try out the idea
+        if self.vmware_test_platform is 'govcsim' or None:
```

Comment:
I'd put this at the end and just make it an else so that it's the default if an invalid (or no option) is chosen.